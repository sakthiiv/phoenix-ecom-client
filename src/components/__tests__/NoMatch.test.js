import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from '../NoMatch';

const location = { pathname: '/product-detail'};
describe('Admin', () => {
    it('renders correctly', () => {
      const wrapper = shallow(<NoMatch location={location}/>);
      expect(wrapper).toMatchSnapshot();
      // On the first run of this test, Jest will generate a snapshot file automatically.
    });
});