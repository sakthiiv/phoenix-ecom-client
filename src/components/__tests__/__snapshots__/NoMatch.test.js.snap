// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin renders correctly 1`] = `
<div>
  <h3>
    No match for 
    <code>
      /product-detail
    </code>
  </h3>
</div>
`;
